### PR TITLE
added cost info to most of the pages where users need to get informed

### PR DESCRIPTION
**Pull request by Milstein**

### CODE REVIEW

File: docs/get-started/allocation/allocation-change-request.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ This will bring up the detailed Quota attributes for that project as shown below
 
 ![Request Change Resource Allocation Attributes for OpenStack Project](images/coldfront-openstack-allocation-attributes.png)
```
Comment (joachimweyl) [inline docs/get-started/allocation/allocation-change-request.md:13]: is this section supposed to have Institution - Specific Code?

File: docs/get-started/cost-billing/how-pricing-works.md [hunk 2]
```diff
@@ -106,6 +106,25 @@ volumes, and you can also create extra volumes yourself. In OpenShift pods, stor
...
+    The **Storage cost** is determined by
+    [your requested and approved allocation values](../allocation/allocation-details.md#pi-and-manager-view).
+    Once approved, these **Storage quotas** will need to be reserved from the
+    total NESE storage pool for both **NERC (OpenStack)** and **NERC-OCP (OpenShift)**
+    resources.
+
+    Even if you have deleted all volumes, snapshots, and object storage buckets and
+    objects in your OpenStack and OpenShift projects. It is very essential to
+    adjust the approved values for your NERC (OpenStack) and NERC-OCP (OpenShift)
+    resource allocations to zero (0) otherwise you still be incurring charge for
```
Comment (joachimweyl) [inline docs/get-started/cost-billing/how-pricing-works.md:119]: you still -> you will still
incurring -> incurring a

File: docs/get-started/cost-billing/how-pricing-works.md [hunk 3]
```diff
@@ -106,6 +106,25 @@ volumes, and you can also create extra volumes yourself. In OpenShift pods, stor
...
+    the approved storage as explained in [Billing FAQs](billing-faqs.md).
+
+    Keep in mind that you can easily scale and expand your current resource
+    allocations within your project. Follow [this guide](../allocation/allocation-change-request.md#request-change-resource-allocation-attributes-for-openstack-project)
```
Comment (joachimweyl) [inline docs/get-started/cost-billing/how-pricing-works.md:123]: not sure if this is another MD issue but I don't see a link here

File: docs/get-started/cost-billing/how-pricing-works.md [hunk 4]
```diff
@@ -106,6 +106,25 @@ volumes, and you can also create extra volumes yourself. In OpenShift pods, stor
...
+    on how to use NERC's ColdFront to reduce your **Storage quotas** for NERC (OpenStack)
+    allocations and [this guide](../allocation/allocation-change-request.md#request-change-resource-allocation-attributes-for-openshift-project)
```
Comment (joachimweyl) [inline docs/get-started/cost-billing/how-pricing-works.md:125]: this link also does not appear to work in MD

**Review by joachimweyl — CHANGES_REQUESTED** [review]
change requests inline.

File: docs/get-started/allocation/allocation-change-request.md (hunk 1 above)
Comment (Milstein) [reply docs/get-started/allocation/allocation-change-request.md:13]: For new screenshot with newly added Attributes I will create a new PR.

File: docs/openstack/backup/backup-with-snapshots.md [hunk 5]
```diff
@@ -257,21 +257,23 @@ below:
...
     of your request, as explained in our
     [Billing FAQs](../../get-started/cost-billing/billing-faqs.md).
 
-    Conversely, if you request a reduction in the **Storage quotas**, specifically
-    the "OpenStack Volume GB Quota" and "OpenStack Swift Quota in Gigabytes", through
-    a [change request using ColdFront](../../get-started/allocation/allocation-change-request.md#request-change-resource-allocation-attributes-for-openstack-project),
+    Conversely, if you request a reduction in the **Storage quotas**, specified
+    by the "OpenStack Volume GB Quota" and "OpenStack Swift Quota in Gigabytes",
+    through a [change request using ColdFront](../../get-started/allocation/allocation-change-request.md#request-change-resource-allocation-attributes-for-openstack-project),
     your invoicing will be adjusted accordingly when the request is approved or
```
Comment (joachimweyl) [inline docs/openstack/backup/backup-with-snapshots.md:277]: invoicing is based on when the adjustment for shrinking is sent, not when it is processed.

**Review by joachimweyl — CHANGES_REQUESTED** [review]
Change requests added inline.

**Review by joachimweyl — APPROVED** [review]

**Pull request merged**